src/App.js

```diff
@@ -1,23 +1,25 @@
-import logo from './logo.svg';
 import './App.css';
+import EntryForm from "./EntryForm"
+import SelectedEntry from './SelectedEntry';
+import OldEntries from './OldEntries';
+import {Routes, Route} from "react-router-dom"
 
 function App() {
+
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <h1>Confidant</h1>
+      <Routes>
+        <Route path ="/" element={
+          <>
+            <EntryForm />
+            <OldEntries />
+          </>
+        }/>;
+        <Route path ="/:entryID" element={<SelectedEntry />}/>;
+        <Route path="*" element={<h2>Error</h2>}/>;
```

Comment:
Nice to see routes in action. Looking forward to one day implementing this in my project too.